mocktest.py

```diff
@@ -0,0 +1,41 @@
+
+from urllib.error import HTTPError
+from sunpy.net import Scraper 
+from sunpy.time import TimeRange
+import logging
+log = logging.getLogger('sunpy')
+log.setLevel('DEBUG') 
+# Set logging level (optional, adjust as needed)
+  # Adjust to DEBUG, INFO, WARNING, etc.
+
+# Add a file handler to save logs (optional)
+with log.log_to_file('logs.log'): 
```

Comment:
What is the goal for the log file?

Reply:
See if https://docs.pytest.org/en/7.1.x/how-to/logging.html helps.

Reply:
Well it depends, if its a logging message we need, then we will assert that the correct message is added to the log.

If the scarper raises an error, then we check that it is raised correctly with the message we expect.